keps/sig-storage/3751-volume-provisioned-io/kep.yaml

```diff
@@ -0,0 +1,44 @@
+title: Kubernetes Volume Provisioned IO
+kep-number: 3751
+authors:
+  - "@mattcarry"
+  - "@sunnylovestiramisu"
+owning-sig: sig-storage
+participating-sigs:
+status: implementable
+creation-date: 2023-01-24
+reviewers:
+  - "@msau42"
+  - "@gnufied"
+approvers:
+  - "@msau42"
+  - "@xing-yang"
+
+see-also:
+replaces:
+
+# The target maturity stage in the current dev cycle for this KEP.
+stage: alpha
+
+# The most recent milestone for which work toward delivery of this KEP has been
+# done. This can be the current (upcoming) milestone, if it is being actively
+# worked on.
+latest-milestone: "1.28"
+
+# The milestone at which this feature was, or is targeted to be, at each stage.
+milestone:
+  alpha: "v1.28"
+  beta: "v1.29"
+  stable: "v1.30"
+
+# The following PRR answers are required at alpha release
+# List the feature gate name and the components for which it must be enabled
+feature-gates:
+  - name: VolumeAttributesClass
+    components:
+      - external-resizer
```

Comment:
I see kube-controller-manager mentioned a few times in the KEP.  Please clarify whether that is needed or not.

Reply:
it is needed for the pvc protection controller